### PR TITLE
Addition of ASVS, CWE, GDPR and MSTG compliances to vendor

### PR DESCRIPTION
**Pull request by rajan262**
1. Moved the existing list of standards inside HIPAA, OWASP and PCIDSS from mycroft to vendor.
2. Added four new compliances: ASVS, CWE, GDPR and MSTG.
3. Updated report.py and report_sample1.json files to reflect new compliances.
4. Moved HIPAA_DATA, OWASP_DATA and PCIDSS_DATA from mycroft to vendor.

### CODE REVIEW

File: ak_vendor/report/report_template.html [hunk 1]
```diff
@@ -1036,6 +1036,78 @@ <h3 class="analysis__sectionheader">{% trans "Regulatory" %}</h3>
...
+                {% if analysis.regulatory.cwe %}
+                  <div class="regulatory__category">
+                    <strong class="regulatory__title">{% trans "CWE" %}</strong>
+                  </div>
+                  <table class="table table--regulatory regulatory__table">
+                    <tbody>
+                      {% for cwe in analysis.regulatory.cwe %}
+                        <tr>
+                          <td>{{ cwe.code }}</td>
+                          <td>{{ cwe.url }}</td>
```
Comment (subho007) [inline ak_vendor/report/report_template.html:1085]: need to have `<a href` defined for urls

**Review by subho007 — CHANGES_REQUESTED** [review]

File: ak_vendor/report/report_template.html (hunk 1 above)
Comment (rajan262) [reply ak_vendor/report/report_template.html]: @subho007 I'm resolving this but please verify at your end as well.

File: ak_vendor/report_sample1.json [hunk 2]
```diff
@@ -113,6 +113,38 @@
...
+          }
+        ],
+        "asvs": [
+          {
+            "code": "V1.1.1",
+            "title": "Verify the use of a secure software development lifecycle that addresses security in all stages of development."
+          },
+          {
+            "code": "V1.1.2",
+            "title": "Verify the use of threat modeling for every design change or sprint planning to identify threats, plan for countermeasures, facilitate appropriate risk responses, and guide security esting."
```
Comment (subho007) [inline ak_vendor/report_sample1.json:130]: spelling mistakes

**Comment by subho007** [comment]
@rajan262 please verify your PR with spelling mistakes. It is god to go if you have fixed any spelling errors/mistakes which might crop up.

File: ak_vendor/report_sample1.json (hunk 2 above)
Comment (rajan262) [reply ak_vendor/report_sample1.json]: I've verified all the places where I copied the content and there is higher chance of spelling mistakes. But the contents in asvs.py is so huge that I can't even check it 😂  but the good thing is it was scraped using a script.

**Review by subho007 — APPROVED** [review]

**Comment by subho007** [comment]
@rajan262 this is approved

**Pull request merged**